main.go

```diff
@@ -47,15 +50,21 @@ func (depper *Depper) registerIngestors() {
 	depper.registerIngestorStream(ingestors.NewNPM())
 	depper.registerIngestor(ingestors.NewElm())
 	depper.registerIngestor(ingestors.NewGo())
-	// depper.registerIngestor(ingestors.NewMavenCentral())
+	depper.registerIngestor(ingestors.NewMavenCentral())
 	depper.registerIngestor(ingestors.NewCargo())
 }
 
 func (depper *Depper) registerIngestor(ingestor ingestors.Ingestor) {
 	c := cron.New()
 	ingestAndPublish := func() {
+		ttl := defaultTTL
+
+		if ttler, ok := ingestor.(ingestors.TTLer); ok {
```

Comment:
This type assertion looks if it's a TTLer and uses the defined TTL, and if not falls back to the default.